spec/lib/rmt/downloader_spec.rb

```diff
@@ -6,6 +6,10 @@
   let(:downloader) { described_class.new(repository_url: 'http://example.com', local_path: dir) }
   let(:headers) { { 'User-Agent' => "RMT/#{RMT::VERSION}" } }
 
+  after do
+    FileUtils.remove_entry(dir)
```

Comment:
We've got FakeFS now in @thutterer's pull request, check it out -- I think it makes sense to use it everywhere.

Reply:
As I ran into lots of small issues while trying this, I don't think that it is worth reworking all of these functioning tests right now.

This would require rewriting specs that deduplication doesn't touch and I feel that this is outside the scope here. I ran into full btrf filesystem issues, so I cleaned up after ourselves for the previous and new tests.

Reply:
OK, maybe I'll refactor it all myself later to use FakeFS.